DSharpPlus/Entities/Channel/Message/DiscordMessage.cs

```diff
@@ -389,6 +397,11 @@ private IMention[] GetMentions()
             mentions.Add(new RepliedUserMention()); // Return null to allow all mentions
         }
 
+        if (this.MessageSnapshots is not null && this.mentionedUsers.Any(r => this.MessageSnapshots.Any(m => r.Id == m.Message?.Author?.Id)))
+        {
+            mentions.Add(new RepliedUserMention()); // Return null to allow all mentions
+        }
+

```

Comment:
I'm unsure if this is needed, couldn't quite understand it. Modeled it after ReferencedMessage. Let me know if I should remove this.

Reply:
Based on what I can understand of the code, and how forwards currently function, forwards cannot ping, so this is probably incorrect

Reply:
Sounds like this block can just be removed then?